README.md

```diff
@@ -201,6 +201,14 @@ For further information on these topics, consult the [module API][30] documentat
 
 For examples, refer to the [development server code][31].
 
+### Styling
```

Comment:
Missed it again - moved to the other document and merge. It's worth another look, if you have time, as I am trying to begin to nudge the reader to use `dist/crosswords.css` rather than `style/crosswords.less` to avoid forcing a less dependency on users. So put these paragraphs under `## Overview` and moved most of the existing docs to `## Internals` - less is now just an internal dependency, not really something an npm-user needs to worry about.

I moved out even the feature from here to there, I want to avoid there being two potentially conflicting docs about styling for now!